examples/api/lib/widgets/safe_area/safe_area.0.dart

```diff
@@ -0,0 +1,288 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+/// Flutter code sample for [SafeArea].
+///
+/// The app is wrapped with [Insets] (defined below)
+/// to simulate a mobile device with a notched screen.
+
+void main() => runApp(const Insets());
+
+class SafeAreaExampleApp extends StatelessWidget {
```

Comment:
I would consider simplify this example a bit since people usually looking for a quick demo without complex setup and focus on the SafeArea only

Reply:
after read through the chat history, seems like the main purpose is to showcase the UI instead of the code. then it looks good to me